tools/editor/editor_import_export.cpp

```diff
@@ -1441,7 +1441,7 @@ bool EditorExportPlatformPC::can_export(String *r_error) const {
 	String err;
 	bool valid=true;
 
-	if (use64 && (!exists_export_template(debug_binary64)) || !exists_export_template(release_binary64)) {
+	if ((use64 && (!exists_export_template(debug_binary64))) || !exists_export_template(release_binary64)) {
```

Comment:
`if (use64 && (!exists_export_template(debug_binary64) || !exists_export_template(release_binary64)))` would be more logical, don't you think?